### PR TITLE
Add motivation section and benchmarks

### PR DESCRIPTION
**Pull request by danieldk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,76 @@
 
 # thinc-apple-ops
 
-Make [thinc](https://thinc.ai) faster on MacOS by calling into Apple's native libraries.
+Make [thinc](https://thinc.ai) faster on macOS by calling into Apple's native
```
Comment (ines) [inline README.md:5]:
```suggestion
Make [spaCy](https://spacy.io) and [Thinc](https://thinc.ai) faster on macOS by calling into Apple's native
```

File: README.md [hunk 2]
```diff
@@ -2,4 +2,76 @@
...
+libraries.
+
```
Comment (ines) [inline README.md:7]: We should probably have a quick summary here or repeat the benchmarks (or a smaller version of them) here so people don't have to scroll to get the juicy numbers?

File: README.md [hunk 3]
```diff
@@ -2,4 +2,76 @@
...
+## 🏫 Motivation
```
Comment (ines) [inline README.md:8]: 🤓🔥👍

File: README.md [hunk 4]
```diff
@@ -2,4 +2,76 @@
...
+Using `thinc-apple-ops` leads to large speedups in prediction and training on
+Apple Silicon Macs, as shown by the benchmarks below.
+
+### Prediction
+
+This first benchark compares prediction speed of the `de_core_news_lg` spaCy
+model between the M1 with and without `thinc-apple-ops`. Results for an Intel
+Mac Mini and AMD Ryzen 5900X are also provided for comparison. Results are in
+words per second. In this prediction benchmark, using `thinc-apple-ops` improves
+performance by *4.3** times.
```
Comment (ines) [inline README.md:59]:
```suggestion
performance by **4.3** times.
```

(Or was this supposed to be italics with `*` for footnote that doesn't exist yet?)

File: README.md [hunk 5]
```diff
@@ -2,4 +2,76 @@
...
+| ------------------------ | ------ | ----------------- |
+| Mac Mini (M1)            | 6492   | 27676             |
+| MacBook Air Core i5 2020 |        |                   |
+| AMD Ryzen 5900X          | 22568  | N/A               |
+
+### Training
+
+In the second benchmark, we compare the training speed of the `de_core_news_lg`
+spaCy model (without NER). The results are in training iterations per second.
+Using `thinc-apple-ops` improves training time by *3.0** times
```
Comment (ines) [inline README.md:71]:
```suggestion
Using `thinc-apple-ops` improves training time by **3.0** times
```

(see other comment)

File: README.md (hunk 4 above)
Comment (danieldk) [reply README.md:59]: It was supposed to be italics, but my editor added one asterisk too many when closing the italics. But maybe bold is even better.

**Comment by adrianeboyd** [comment]
The CI results are worrying me here? Nothing changed since: https://github.com/explosion/thinc-apple-ops/runs/3891919224

**Comment by adrianeboyd** [comment]
Ah, how it sets though "cpu" is different and goes through the registry, when it didn't before. Hmm, this is not going to work...

**Pull request merged**